src/main/docker/Dockerfile

```
@@ -1,4 +1,11 @@
-FROM openjdk:17-jdk-slim-bullseye
+FROM eclipse-temurin:17 as builder
+WORKDIR /build
+COPY . .
+RUN chown -R 1000.1000 /build
```

```suggestion
RUN chown -R 1000:1000 /build
```
This is definitely an error. I also think npm tries to store the npm cache in `$HOME/.npm/`. I’m not sure how that is handled if user 1000 does not have a `$HOME`-dir.